lib/kamal/commands/builder/pack.rb

```diff
@@ -0,0 +1,34 @@
+class Kamal::Commands::Builder::Pack < Kamal::Commands::Builder::Base
+  def push
+    combine \
+      pack(:build,
+        config.repository,
+        "--platform", platform,
+        "--builder", pack_builder,
+        buildpacks,
+        "-t", config.absolute_image,
+        "-t", config.latest_image,
+        "--env", "BP_IMAGE_LABELS=service=#{config.service}",
+        *argumentize("--env", args),
+        *argumentize("--env", secrets, sensitive: true),
+        "--path", build_context),
+      docker(:push, config.absolute_image),
+      docker(:push, config.latest_image)
+  end
+
+  def remove;end
```

Comment:
We're not actually creating anything with the buildpack setup so there isn't anything to remove. Should we still `puts` something out to the user?